File: pset1/mario/mario.py
from cs50 import get_int

def main():

    while True:
        h = get_int("number in range of 1 to 8: ")
        if h >= 0 and h <= 9:
            break

    for a in range(h):

        for l in range(h-a):
            print(" ", end="")

        for d in range(h-a, h + 1):
            print("#", end="")

        print(" ", end="")

        for k in range(h-a, h + 1):
            print("#", end="")

        print("")

if __name__ == "__main__":
    main()

File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main (void)
{
    int height;
    int c = 0;

    while(height < 1 || height > 8) {
        printf("Welche Hoehe?\n");
        height = get_int();
    }

    for(int i = height; i > 0; i--) { // Höhe
        for(int space = 0; space < i - 1; space++) { //Leerzeichen vor #
            printf(" ");
        }

        printf("#"); //Gibt ein # aus
        c++;

        for (int hash = 0; hash < c - 1; hash++) { //Gibt zusätzliche # aus
            printf("#");
        }

        printf(" "); // Leerzeichen

        for (int hash = 0; hash < c; hash++) { //Gibt zusätzliche # aus
            printf("#");
        }
        printf("\n");
    }
}
